code/game/machinery/syndicatebomb.dm

```diff
@@ -490,9 +490,9 @@
 			for(var/obj/item/slime_extract/S in LG.beakers) // And slime cores.
 				if(beakers.len < max_beakers)
 					beakers += S
-					S.loc = src
+					S.forceMove(src)
 				else
-					S.loc = get_turf(src)
+					S.drop_location()
```

Comment:
don't you need to force move it to the drop location?

Reply:
Yes, this should be S.forceMove(drop_location()). And you do this 999x times. Fix your shit @kevinz000 !